src/test/skript/tests/syntaxes/expressions/ExprFreezeTicks.sk

```diff
@@ -0,0 +1,17 @@
+test "freeze time" when running minecraft "1.18":
+	spawn cow at location(0, 64, 0, world "world")
```

Comment:
The test world is a super flat, The cow will die from fall damage at y 64 lol.

You can simply use the `spawn of world "world"` instead of the location function.

I would also suggest utilizing an effect section for this spawn.
```vb
spawn cow at spawn of world "world":
    #do tests with event-entity
```